src/Benchotron/Utils.purs

```diff
@@ -13,5 +13,7 @@ toAny = mkExists <<< Identity
 foreign import unsafeJsonStringify ::
   forall a. a -> String
 
-(>>) :: forall m a b. (Bind m) => m a -> m b -> m b
-(>>) x y = x >>= const y
+bindConst :: forall m a b. (Bind m) => m a -> m b -> m b
```

Comment:
Not a nitpick at all - no need to duplicate something that already exists. I'm guessing kika missed the comment on my initial PR. I'm pulling this now, and I'll move to `*>`. 👍